guides/micronaut-cloud-oidc-base/start.adoc

```diff
@@ -0,0 +1,6 @@
+common:header.adoc[]
+
+The application will use OpenID Connect (OIDC), which is an identity layer on top of the OAuth 2.0 protocol, allowing verification of the identity of an end user and obtaining basic profile information.
+Under the hood, the OIDC APIs are easier to work with than OAuth, using JSON instead of XML, but in practice Micronaut Security handles the details for you.
```

Comment:
```suggestion
Under the hood, the OIDC APIs are easier to work with than OAuth because they use JSON instead of XML, but in practice, Micronaut Security handles the details for you.
```